src/navigation/navigation.d.ts

```diff
@@ -19,6 +19,7 @@ type DrawerParamList = {
   RegistrarList: undefined;
   Webview: {uri: string; title: string};
   DevScreen: undefined;
+  PushSettingsScreen: undefined;
```

Comment:
How about `NotificationSettingsScreen` ?  It's kind of a nitpick

Reply:
done ✅